grpc/server/src/CMakeLists.txt

```diff
@@ -1,67 +1,33 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.1)
 
-file(STRINGS ${PLUGINS_DIR}/plugins.conf PLUGINS_LIST)
+set(COMPONENTS_LIST core action mission telemetry)
```

Comment:
Oh! No plugin list ?

Reply:
For now, it is hard-coded. We need to deliver a first version soon. So our first version will not really support plugins (i.e. the backend has all our plugins). Later, we'll upgrade for a more dynamic architecture.

Still, the frontend now should look like the plugins are loaded dynamically, so that it is transparent to the user when we update the backend :-).

Reply:
OK :-)